features/providers/non_passported_journey.feature

```diff
@@ -306,52 +316,6 @@ Feature: Non-passported applicant journeys
     And I should not see 'What happens next'
     But I should see 'Your client needs to complete their part of the application before you can continue.'
 
```

Comment:
I removed this as the provider now adds student finance as part of the means questions so this doesn't need to be tested separately